tigerpath/majors_and_certificates/majors/MUS_2018.json

```
@@ -0,0 +1,165 @@
+{
+  "type": "Major",
+  "name": "Music",
+  "code": "MUS",
+  "degree": "AB",
+  "year": 2018,
+  "explanation": "The Department of Music aims to give students a broad foundation for making, performing, studying, and writing about music. As part of the major, students pursue work in two loosely defined areas: Culture and Criticism (C&C) and Materials and Making (M&M). M&M encompasses topics usually explored in hands-on, exploratory ways, such as composition, improvisation, theory, analysis, and electronic music. The C&C area focuses on the scholarly study of music, from historical and formalist analysis to cognitive science and ethnography. Whether our concentrators ultimately choose to make music, to research and write about it, or do both, they are encouraged to develop independent work that moves across traditional disciplinary or methodological boundaries.",
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-music",
+    "https://music.princeton.edu/program-study"

```

Also possibly useful
```suggestion
    "https://music.princeton.edu/program-study",
    "https://music.princeton.edu/sites/default/files/ua_department_of_music_2020.pdf"
```